docs/UserGuide.md

```diff
@@ -274,7 +274,7 @@ Format: `import`
 </box>
 
 * If the program comes across a contact that has an empty or invalid contact info, it will attempt to fill in the info
-* It will fill in the info with placeholder values (0000 0000, default@email.net, @DefaultHandle)
+* It will fill in the info with placeholder values (0000 0000, default@email.net, @DefaultHandle, AA0000)
```

Comment:
The default telegram handle in your JsonAdaptedPerson.java seems to be @defaulthandle but over here the D and H are capitalised. Does the space between the 0s for phone number affect anything as the one in JsonAdaptedPerson.java does not seem to have any spaces?

Reply:
space has no impact on anything other than identifying it as having 8 digits, as is the common format for many phone numbers